service_container/tags.rst

```diff
@@ -692,6 +692,11 @@ iterator, add the ``exclude`` option:
 
     The ``exclude`` option was introduced in Symfony 6.1.
 
+.. versionadded:: 6.3
```

Comment:
When tagging 7.0 all versionadded directives will be drop, so maybe it should be a note/tip + versionadded ?

Reply:
Yes that would be better 👍

Reply:
That was my main concern about this PR. Updated it with a note+versionadded, thank you both!